common/src/app/common/data.cljc

```diff
@@ -229,6 +229,11 @@
   #?(:clj (Object.)
      :cljs (js/Object.)))
 
+(defn getf
+  "Returns a function to access a map"
+  [coll]
+  (partial get coll))
```

Comment:
As explained in other comment, prefer reader macro if you can over `partial`, the `partial` has performance implications.